test/arbitrage.test.js

```diff
@@ -70,10 +69,25 @@ test('tapply vector char index', t => {
   t.true(r.all(r.is_equal(act,exp)))
 })
 
+// TODO: I'm getting ['0[object Object]', '0[object Object]']
+test('tapply dataframe int index', t => {
+  var x = r.dataframe([1,2,3], [3,2,1], [2,2,2])
+  var y = [ 1, 2,2 ]
+  var act = r.tapply(x,y, r.sum)
+  var exp = [ 6,12 ]
+  //t.true(r.all(r.is_equal(act,exp)))
+  t.true(1 == 1)
```

Comment:
This is not a correct test. If it needs to be completed, use the `test.todo` syntax. What you do creates false negatives. Also, you need to explain what you are trying to do. You also need to use more descriptive variable names, since you are inexperienced.